cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java

```diff
@@ -147,6 +148,11 @@ public String getFormat() {
             ParameterType.BOOLEAN,
             "Export power flows for switches",
             Boolean.FALSE);
+    private static final Parameter KEEP_TOPOLOGICAL_MAPPING_PARAMETER = new Parameter(
```

Comment:
`KEEP` is maybe not needed.

Reply:
Do you prefer `WITH_TOPOLOGICAL_MAPPING`?

Reply:
Yes.